cegs_portal/search/templates/search/v1/experiment.html

```diff
@@ -348,10 +352,348 @@
         role="tabpanel"
         aria-labelledby="tabs-qc-graphs-tab"
     >
-        <div class="datafile-content-section content-container min-w-max max-w-max">QC plots TK</div>
+        <div
+            class="datafile-content-section content-container min-w-max max-w-max text-xl font-bold"
+            style="color: gray"
+        >
+            QC Graphs <i class="bi bi-file-bar-graph-fill"></i>
+            <div class="faded-line my-3.5"></div>
+            <div class="flex flex-nowrap my-3.5">
+                <div id="ncells_histogram" class="mr-4"></div>
+                <div id="ngrna_histogram"></div>
+            </div>
+            <div class="flex flex-nowrap my-3.5">
+                <div id="uniform_qqplot" class="mr-4"></div>
+                <div id="volcano"></div>
+            </div>
+        </div>
     </div>
 </div>
+
 {% endblock %} {% block inline_javascript %}
+
+<script>
+    // setup API options
+    const options = {
+        config: {
+            // Vega-Lite default configuration
+        },
+        init: (view) => {
+            // initialize tooltip handler
+            view.tooltip(new vegaTooltip.Handler().call);
+        },
+        view: {
+            // view constructor options
+            // remove the loader if you don't want to default to vega-datasets!
+            loader: vega.loader({
```

Comment:
What does this do? You use it here and in the QQ plot, but not in the histograms.